x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -33,12 +33,12 @@ query
     ;
 
 sourceCommand
-    : explainCommand
-    | fromCommand
+    : fromCommand
     | rowCommand
     | showCommand
     // in development
     | {this.isDevVersion()}? timeSeriesCommand
+    | {this.isDevVersion()}? explainCommand
```

Comment:
Moving it to dev mode. It didn't work before, so there is really no reason to have it in the GA grammar